### PR TITLE
Fix ArgoCD Redis HA Sa permissions.

### PR DESCRIPTION
**Pull request by jtudelag**
#### What is this PR About?
Describe the contents of the PR

#### How do we test this?
Provide commands/steps to test this PR.

cc: @redhat-cop/day-in-the-life

### CODE REVIEW

**Comment by jtudelag** [comment]
Fixes: https://github.com/redhat-cop/helm-charts/issues/190

File: charts/argocd-operator/templates/anyuid-scc.yaml [hunk 1]
```diff
@@ -1,29 +1,16 @@
...
+  name: argocd-redis-ha-anyuid-scc
+  namespace: {{ .Values.namespace }}
 roleRef:
   apiGroup: rbac.authorization.k8s.io
   kind: ClusterRole
-  name: {{ .Values.namespace }}-argocd-redis-ha-anyuid-scc
+  name: system:openshift:scc:anyuid
 subjects:
 - kind: ServiceAccount
-  name: {{ .Values.name }}-argocd-redis-ha
```
Comment (ckavili) [inline charts/argocd-operator/templates/anyuid-scc.yaml:27]: the first `argocd` is coming from ArgoCD CR name which is default to `argocd` in this chart. If anyone would like to choose a different name, it'll be different for them. Therefore it should stay as `{{ .Values.name }}-argocd-redis-ha` 🙌🏻

**Review by ckavili — CHANGES_REQUESTED** [review]

File: charts/argocd-operator/templates/anyuid-scc.yaml (hunk 1 above)
Comment (jtudelag) [reply charts/argocd-operator/templates/anyuid-scc.yaml:27]: done ;)

**Comment by ckavili** [comment]
thanks @jtudelag 🚀

**Pull request merged**